### PR TITLE
Implement addField for Aggregate

### LINKED ISSUE #40: How to use `$addFields` aggregation
First of all, thanks for your work on this library, We started to use it in [lichess.org](https://github.com/lichess-org/lila-seach) and it's just wonderful to use.

I just wonder how to use [`$addFields` aggregation](https://www.mongodb.com/docs/manual/reference/operator/aggregation/addFields/) with mongo4cats. I see that We support some similar operation like `unwind`, `replaceWith` but not `addFields`.

I can create `Document`, `Bson` by my own, but I couldn't find a way to create an `Aggregate` our of them.

So, imho, We could solve this by two ways:
- support `$addFields` and some other missing aggregations out of the box
- support a way to construct aggration from `org.boson.Bson` or `BsonValue`.

I'm willing to create a PR if you think it's a good idea?

### PR DESCRIPTION
**Pull request by lenguyenthanh**
close #40 

Couldn't help my self to implement it before your answer, as I need to use this in our project.

p/s: It works in my machine

### CODE REVIEW

File: modules/kernel/src/main/scala/mongo4cats/operations/Aggregate.scala [hunk 1]
```diff
@@ -223,6 +224,20 @@ trait Aggregate extends AsJava {
...
+    *
+    * <p>With \$addFields, you can adds the new fields the document.</p>
+    *
+    * @param name
+    *   the name of the new field
+    * @param value
+    *   the value of the new field
+    * @return
+    *   the \$addFields pipeline stage [[https://www.mongodb.com/docs/manual/reference/operator/aggregation/addFields/]]
+    * @since 3.4
```
Comment (lenguyenthanh) [inline modules/kernel/src/main/scala/mongo4cats/operations/Aggregate.scala:237]: I got the version from: https://www.mongodb.com/docs/v3.4/reference/operator/aggregation/addFields/

**Comment by Kirill5k** [comment]
Looks good, however could you please change the name of the method from `addFields` to `addField`, since you are only adding a single field in there?

**Comment by Kirill5k** [comment]
Looking good. thank you very much 👍
I'll release it today

**Comment by lenguyenthanh** [comment]
I updated so now We have two versions of `addFields` similar to facet which allow varargs or List as arguments.

oops you merged it when I'm writing this comment. Thanks!

**Pull request merged**